test/test_helper.rb

```diff
@@ -1,3 +1,6 @@
+require 'coveralls'
+Coveralls.wear!
+
```

Comment:
I had planned to allow using RSpock as a development dependency for ASTTransform, and I remember there were some quirks around where to install the hook, I ended up creating a `test_loader.rb` file that is loaded from the rake `test` task, which then `require 'ast_transform'`, which makes it so SimpleCov doesn't report coverage, since it needs to be required and the call to `SimpleCov.start` must be done before any application code is loaded.

Unless we prevent using RSpock in ASTTransform and revert to a more traditional `test` rake task, this should be done in `test_loader.rb`, as is done on this branch: #3